### PR TITLE
SAK-47131 Menu items in assignments tool are too close and need some separation

### PR DESCRIPTION
**Pull request by PabloArrz**
In Sakai 22, in Assignments tool, when a user is going to grade submissions the menu items (“Download All“,”Upload All” and ”Release Grades”) are too close. A little separation is needed

### CODE REVIEW

**Comment by mpellicer** [comment]
Thanks @PabloArrz, 

Could you please sign the Apereo ICLA here?  I don't see you listed.

https://www.apereo.org/licensing/agreements/icla

I don't know how many times I made changes in these links, I don't like the old nor the new UI, I believe we can spend time on this and improve it a little bit, thanks!

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm [hunk 1]
```diff
@@ -120,8 +120,8 @@ function printView(url) {
 							<a href="#toolLinkParam("$action" "doPrep_download_all" "view=$!view")" class="button" id="downloadAll" title="$!tlang.getString('downall')">$!tlang.getString('downall')</a>
 						#if(!$disableGrade)
 							## upload all
-							<a href="#toolLinkParam("$action" "doPrep_upload_all" "view=$view")" class="button" id="uploadAll" title="$!tlang.getString('uploadall.title')">$!tlang.getString('uploadall.title')</a>
-							<a href="#"
+							| <a href="#toolLinkParam("$action" "doPrep_upload_all" "view=$view")" class="button" id="uploadAll" title="$!tlang.getString('uploadall.title')">$!tlang.getString('uploadall.title')</a>
+							| <a href="#"
```
Comment (fostersdesign) [inline assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm:124]: Screen readers will read this as "vertical pipe" in between each link, which is noise for those users. I'd suggest adding these as borders instead.

**Review by fostersdesign — CHANGES_REQUESTED** [review]

**Comment by PabloArrz** [comment]
Okey perfect, finally I use CSS in the vertical pipes.

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm [hunk 2]
```diff
@@ -819,7 +819,6 @@ function printView(url) {
 			}
 		});
 
-		ASN.enableLinks();
```
Comment (V100X) [inline assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm:822]: Hi, is this method on [assignments.js](https://github.com/sakaiproject/sakai/blob/07ce149f5081fc26d29dca6a956528e4343901c9/assignment/tool/src/webapp/js/assignments.js) still relevant?

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm [hunk 3]
```diff
@@ -117,12 +117,12 @@ function printView(url) {
...
 				<div class="listNav">
 						## download all
-							<a href="#toolLinkParam("$action" "doPrep_download_all" "view=$!view")" class="button" id="downloadAll" title="$!tlang.getString('downall')">$!tlang.getString('downall')</a>
+							<a href="#toolLinkParam("$action" "doPrep_download_all" "view=$!view")" class="AssignmentItem" id="downloadAll" title="$!tlang.getString('downall')">$!tlang.getString('downall')</a>
 						#if(!$disableGrade)
 							## upload all
-							<a href="#toolLinkParam("$action" "doPrep_upload_all" "view=$view")" class="button" id="uploadAll" title="$!tlang.getString('uploadall.title')">$!tlang.getString('uploadall.title')</a>
-							<a href="#"
-								class="button"
+							 <a href="#toolLinkParam("$action" "doPrep_upload_all" "view=$view")" class="AssignmentItem" id="uploadAll" title="$!tlang.getString('uploadall.title')">$!tlang.getString('uploadall.title')</a>
```
Comment (mpellicer) [inline assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm:123]: Why are we changing the class? I remember proposing "listNav a" as a selector

**Review by mpellicer — CHANGES_REQUESTED** [review]

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm (hunk 3 above)
Comment (mpellicer) [reply assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm:123]: Revert the first white space please, thanks

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm (hunk 2 above)
Comment (mpellicer) [reply assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm:822]: It doesn't make any sense to me.

**Review by mpellicer — CHANGES_REQUESTED** [review]

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm (hunk 1 above)
Comment (mpellicer) [reply assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm]: That's why I removed them in the past!

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm (hunk 3 above)
Comment (PabloArrz) [reply assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm]: Okey now it's done.

**Comment by mpellicer** [comment]
Thanks for the CLA

image

**Pull request merged**